panther_manager/src/manager_bt_node.cpp

```
@@ -101,8 +101,12 @@ ManagerBTNode::ManagerBTNode(
 
   if (launch_lights_tree) {
     const std::map<std::string, std::any> lights_initial_bb = {
+      {"battery_percent", std::numeric_limits<float>::quiet_NaN()},
```

create_bt_config() should throw an exception because it can't cast nan in bb

---

I reverted those changes